libraries/SITL/SIM_Airspeed_DLVR.cpp

```diff
@@ -0,0 +1,65 @@
+#include "SIM_Airspeed_DLVR.h"
+
+int SITL::Airspeed_DLVR::rdwr(I2C::i2c_rdwr_ioctl_data *&data)
+{
+    struct I2C::i2c_msg &msg = data->msgs[0];
+    if (msg.flags == I2C_M_RD) {
+        // driver is attempting to receive reading...
+        if (msg.len != 4) {
+            AP_HAL::panic("Unexpected message length (%u)", msg.len);
+        }
+
+        union PACKED {
+            uint32_t bytes;
+            struct {
+                uint32_t filler : 5;
+                uint32_t temperature_raw : 11;
+                uint32_t pressure_raw : 14;
+                uint32_t status : 2;
+            } exploded;
+        } stuff;
+
+        // this is TYPE_I2C_DLVR_5IN
+
+        stuff.exploded.status = 0;
```

Comment:
If the sim time is stupidly close then this is actually an error, it would be stale data which is `0b10`. Debatable if you want to HAL panic on that or just throw it back at the driver. Also if airspeed fail is set then returning `0b11` is correct.

Reply:
We now populate the status bit appropriately.

Given the sensor only updates @20Hz ATM, it gets set rather frequently.